core/src/cli/cli.ts

```
@@ -285,12 +285,29 @@ ${renderCommands(commands)}
 
     // Init Cloud API
     let cloudApi: CloudApi | null = null
+    let distroName: string = ""
+
     if (!command.noProject) {
       const config: ProjectResource | undefined = await this.getProjectConfig(workingDir)
 
-      let cloudDomain: string = getGardenCloudDomain(config)
+      const cloudDomain: string = getGardenCloudDomain(config)
+      distroName = getCloudDistributionName(cloudDomain)
 
-      cloudApi = await CloudApi.factory({ log, cloudDomain })
+      try {
+        cloudApi = await CloudApi.factory({ log, cloudDomain })
+      } catch (err) {
+        if (err instanceof EnterpriseApiTokenRefreshError) {
+          log.warn(dedent`
```

Do we also just show a warning in projects where the cloud url is configured? This might lead to confusing error messages, e.g. missing secrets

---

We do. 

I suppose you could argue that if you get a token refresh error and cloud domain is set then it should fail hard like before.